evmbin/src/info.rs

```diff
@@ -100,51 +100,58 @@ pub fn run_action<T: Informant>(
 	})
 }
 
+/// Input data to run transaction
+#[derive(Debug)]
+pub struct TxInput<'a, T> {
+	/// Chain specification name associated with the transaction.
+	pub name: &'a String,
```

Comment:
```suggestion
	pub name: &'a str,
```

`&'a str` is less restrictive and will most likely suffice here.